src/MatBlazor/Components/MatTable/BaseMatTable.cs

```diff
@@ -159,11 +157,51 @@ public string SearchTermFieldPlaceHolder
         [Parameter]
         public bool Striped { get; set; }
 
+        private int _pageSize = 5;
```

Comment:
Won't this disable the ability to use the paginator and set the page size?

Reply:
Hi. Not at all.
I made the change so that the page size could be bound to so it would stop resetting to 5 with each button click on the table.
Both the paginator and page size still work with the change (that's how I use the table).
Also, the default page size 5 was pre-existing.

Reply:
Hi. Not at all.
I made the change so that the page size could be bound to so it would stop resetting to 5 with each button click on the table.
Both the paginator and page size still work with the change (that's how I use the table).
Also, the default page size 5 was pre-existing.